### ISSUE TITLE
Invisible icons

### ISSUE THREAD

**Issue by sfwtn** [aichingm/lonewolf#1]
Hey, just saw this project on Flathub and decided to check it out, upon loading in a lot of icons appear to be either invisible or of the same color as the background. Any ideas on what's the problem here?
 
![image](https://github.com/aichingm/lonewolf/assets/121581681/650a02ca-7843-457a-bd9c-9ea0f1c3185a)
![image](https://github.com/aichingm/lonewolf/assets/121581681/20e2cdac-c51b-46ac-bb7c-53c15c8d4795)
![image](https://github.com/aichingm/lonewolf/assets/121581681/910d8693-73fb-46d8-b7b4-3d919eed0722)

Some system info:
```
Operating System: Arch Linux
KDE Plasma Version: 5.27.8
KDE Frameworks Version: 5.110.0
Qt Version: 5.15.10
Kernel Version: 6.1.55-1-lts (64-bit)
Graphics Platform: X11
Flatpak: 1.15.4
```

**Comment by aichingm** [comment]
I think I figured out whats going on. Could it be that you started Lonewolf without internet access? It seams that the icons are downloaded loaded after the first launch. Can you verify this?

This is an issue and should not work this way! Icons should be available without access to third parties. Thanks for reporting!

**Comment by sfwtn** [comment]
Yup, that fixed the problem. I have my flatpak setup to deny Internet by default. Thanks for looking into it!

**Comment by aichingm** [comment]
I published version 1.0.4 on flathub which includes a fix for this issue.
